examples/configs/chapman/initial_conditions.csv

```diff
@@ -0,0 +1,24 @@
+CONC.M,0.779127760953859
+CONC.Ar,0.963825315875336
+CONC.CO2,0.846014255793546
+CONC.H2O,0.084410107675724
+CONC.N2,0.300322986940341
+CONC.O1D,0.925827943758015
+CONC.O,0.566253459937848
+CONC.O2,0.566253459937848
+CONC.O3,0.566253459937848
+ENV.temperature,287.45
```

Comment:
Do we want to use environmental conditions for the stratosphere, say around z = 30 km?

Reply:
that would make slightly more sense for chapman